content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Control Topology Management Policies on a node
+reviewers:
+- ConnorDoyle
+- klueska
+- lmdaly
+- nolancon
+
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+An increasing number of systems leverage a combination of CPUs and hardware accelerators to support latency-critical execution and high-throughput parallel computation. These include workloads in fields such as telecommunications, scientific computing, machine learning, financial services and data analytics. Such hybrid systems comprise a high performance environment.
+
+In order to extract the best performance, optimizations related to CPU isolation and memory and device locality are required. However, in Kubernetes, these optimizations are handled by a disjoint set of components.
+
+_Topology Manager_ is a component in Kubelet that provides node level policies to enable these performance optimizations in an user abstract manner.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Topology Manager
+
+Prior to the introduction of Topology Manager, the CPU and Device Manager in Kubernetes make resource allocation decisions independently of each other.
+This can result in sub-optimal allocations on multiple-socketed systems, performance/latency sensitive applications will suffer due to these sub-optimal allocations. 
+* Sub-optimal in this case meaning for example, CPUs and devices being allocated from different NUMA Nodes those incurring additional latency.
+
+The Topology Manager is a new Kubelet component, that acts as a source of truth for other Kubelet components to make topology aligned resource decisions.
+
+The Topology Manager provides an interface for components, called *Hint Providers*, to send and receive topology information. The default *best-effort* algorithm takes
+all possible NUMA Node combinations for an incoming container and chooses the best fit (being the narrowest NUMA node with resources available). 

```

Comment:
“Node” means something specific in Kubernetes
```suggestion
all possible NUMA node combinations for an incoming container and chooses the best fit (being the narrowest NUMA node with resources available). 
```

(and also elsewhere in the page)